src/go/rdctl/cmd/snapshotList.go

```diff
@@ -81,10 +82,25 @@ func tabularOutput(snapshots []snapshot.Snapshot) error {
 		return nil
 	}
 	writer := tabwriter.NewWriter(os.Stdout, 0, 4, 4, ' ', 0)
-	fmt.Fprintf(writer, "NAME\tCREATED\n")
+	fmt.Fprintf(writer, "NAME\tCREATED\tDESCRIPTION\n")
 	for _, aSnapshot := range snapshots {
 		prettyCreated := aSnapshot.Created.Format(time.RFC1123)
-		fmt.Fprintf(writer, "%s\t%s\n", aSnapshot.Name, prettyCreated)
+		desc := aSnapshot.Description
+		idx := strings.Index(desc, "\n")
+		if idx >= 0 {
+			// If the description starts with a newline, it will appear empty in this view.
+			// Use the json view to get the full description
+			desc = desc[0:idx]
```

Comment:
I'm not sure if it wouldn't be better to just replace non-printing character in the description by spaces, trim leading and trailing spaces, and replace all runs of multiple spaces with a single space. And then treat all descriptions the same.

This would provide more information in the CLI output; the newlines are really only for the GUI code.

Reply:
Should be part of issue 5782 to deal with the cosmetic appearance of a description field in non-json output. Otherwise this PR will never get in.